macos/Classes/DynamicColorPlugin.swift

```diff
@@ -0,0 +1,28 @@
+import Cocoa
+import FlutterMacOS
+
+public class DynamicColorPlugin: NSObject, FlutterPlugin {
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(name: "io.material.plugins/dynamic_color", binaryMessenger: registrar.messenger)
+    let instance = DynamicColorPlugin()
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    switch call.method {
+    case "getControlAccentColor":
+      var nsColor = NSColor.systemBlue
+      if #available(macOS 10.14, *) {
+        nsColor = NSColor.controlAccentColor
+      }
+      var r: CGFloat = 0
+      var g: CGFloat = 0
+      var b: CGFloat = 0
+      var a: CGFloat = 0
+      nsColor.usingColorSpace(.sRGB)!.getRed(&r, green: &g, blue: &b, alpha: &a)
+      result(Int(a * 255) << 24 + Int(r * 255) << 16 + Int(g * 255) << 8 + Int(b * 255))
```

Comment:
maybe better to float array here anyway but not sure if Windows counterpart maybe come later will use that or not. Some multiply it to `*255.99999` https://gist.github.com/gobijan/d724de27e2aff8131676 and some don't https://stackoverflow.com/questions/28644311/how-to-get-the-rgb-code-int-from-an-uicolor-in-swift

Reply:
Oh that's nasty, let's not do that

Reply:
OK